modules/oe_showcase_multilingual/tests/src/ExistingSiteJavascript/MultilingualTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_showcase_multilingual\ExistingSiteJavascript;
+
+use Drupal\Tests\oe_showcase\ExistingSiteJavascript\ShowcaseExistingSiteJavascriptTestBase;
+
+/**
+ * Tests oe_showcase_multilingual translation is available.
+ */
+class MultilingualTest extends ShowcaseExistingSiteJavascriptTestBase {
+
+  /**
+   * Create a page node and test language switcher block.
+   */
+  public function testOeMultilingualTranslation(): void {
+    // Mark test content for deletion after the test has finished.
+    $this->markEntityTypeForCleanup('node');
+
+    $assert_session = $this->assertSession();
+    $page = $this->getSession()->getPage();
+
+    $this->drupalLogin($this->createUser([
+      'create oe_showcase_page content',
+      'delete any oe_showcase_page content',
+      'delete own oe_showcase_page content',
+      'edit own oe_showcase_page content',
+      'translate oe_showcase_page node',
+      'administer content translation',
+      'create content translations',
+      'delete content translations',
+      'update content translations',
+    ]));
+
+    // Create a node and assert we can see all languages in the list but
+    // no operations.
+    $values = [
+      'type' => 'oe_showcase_page',
+      'title' => 'Test page',
+      'status' => 1,
+    ];
+    $node = $this->createNode($values);
+    $this->drupalGet('/node/' . $node->id() . '/translations');
+
+    // Assert all 24 EU languages are available.
+    $rows = $page->findAll('css', 'table tbody tr');
+    $this->assertCount(24, $rows);
+    $assert_session->linkNotExists('Translate locally');
+    $this->assertCount(23, $page->findAll('xpath', '//a[text()=\'Add\']'));
+
+    // Add a translation.
+    $node->addTranslation('pt-pt', ['title' => 'Translated to PT'] + $node->toArray());
+    $node->save();
+    $this->drupalGet($node->toUrl());
+    $this->clickLink('English');
+    $this->getSession()->wait(10000);
```

Comment:
```suggestion
    $this->assertSession()->waitForElementVisible('css', '#languageModal');
```
the idea was to wait for the modal to be visible right ?